view/frontend/requirejs-config.js

```diff
@@ -1,5 +1,7 @@
 var config = {
-    "paths": {
-        "InstagramWidget": "Bitbull_InstagramWidget/js/InstagramWidget"
+    map: {
+        '*': {
+            InstagramWidget: 'Bitbull_InstagramWidget/js/InstagramWidget'
```

Comment:
According to devdocs, this is how it should be implemented in requirejs config.
See:
http://devdocs.magento.com/guides/v2.1/javascript-dev-guide/javascript/custom_js.html